runtime/vscode/lsp/buf.lua

```diff
@@ -0,0 +1,74 @@
+local vscode = require("vscode")
```

Comment:
1 or 2 lines at the top of this file explaining its purpose ?

    -- Shim vim.lsp calls to vscode equivalent functions.